modules/signing_worker/files/update-requirements.sh

```diff
@@ -1,11 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-if [ -z "$1" ]; then
-    echo "Usage: $0 worker_type"
-    echo "Worker type is ff-prod, tb-prod, or dep"
-    echo "Pinned requirements will be output to stdout"
-    exit 1
-fi
+# This script will create a docker image on the local machine with name "pip-compile"
+# It is safe to delete such image after running this script
+# It will also fetch yq and python images which can be safely removed from the local image cache
+
+# Configuration
+yq_docker_image=mikefarah/yq:4.28.2
+python_docker_image=python:3.8.3-alpine3.12
```

Comment:
Big 👍 for making this script reproducible across all types of setups! If by any chance you would like to go one step further, we can even pin the docker image by hash:

```suggestion
python_docker_image=python:3.8.3-alpine3.12@sha256:6c1b18373c4f94353308097772e97ff4d0e596ac3d06cc4b00c3bdc52cd5e8b6
```

https://hub.docker.com/layers/library/python/3.8.3-alpine3.12/images/sha256-6c1b18373c4f94353308097772e97ff4d0e596ac3d06cc4b00c3bdc52cd5e8b6?context=explore